drivers/unix/ip_unix.h

```diff
@@ -38,7 +38,7 @@
 class IP_Unix : public IP {
 	GDCLASS(IP_Unix, IP);
 
-	virtual IP_Address _resolve_hostname(const String &p_hostname, IP::Type p_type);
+	virtual void _resolve_hostname(List<IP_Address> &r_addresses, const String &p_hostname, Type p_type = TYPE_ANY) const override;
```

Comment:
The `3.x` branch doesn't use the `override` keyword, it was added in `master` when modernizing the codebase. So this makes it fail building: https://github.com/godotengine/godot/runs/2773796280
```

Reply:
I see. Removed!